htdocs/libraries/icms/messaging/EmailHandler.php

```diff
@@ -1,157 +1,165 @@
+// Copyright (c) 2000 XOOPS.org //
+// <http://www.xoops.org/> //
+// ------------------------------------------------------------------------ //
+// This program is free software; you can redistribute it and/or modify //
+// it under the terms of the GNU General Public License as published by //
+// the Free Software Foundation; either version 2 of the License, or //
+// (at your option) any later version. //
+// //
+// You may not change or alter any portion of this comment or credits //
+// of supporting developers from this source code or any supporting //
+// source code which is considered copyrighted (c) material of the //
+// original comment or credit authors. //
+// //
+// This program is distributed in the hope that it will be useful, //
+// but WITHOUT ANY WARRANTY; without even the implied warranty of //
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the //
+// GNU General Public License for more details. //
+// //
+// You should have received a copy of the GNU General Public License //
+// along with this program; if not, write to the Free Software //
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA //
+// ------------------------------------------------------------------------ //
+// Author: Jochen Büînagel (job@buennagel.com) //
+// URL: http://www.xoops.org //
+// Project: The XOOPS Project //
 // ------------------------------------------------------------------------- //
 /**
  * Class for handling email, extending PHPMailer to email the users
  *
- * @category	ICMS
- * @package		Messaging
- * @subpackage	Email
- * @copyright	(c) 2007-2008 The ImpressCMS Project - www.impresscms.org
- * @license		http://www.gnu.org/licenses/old-licenses/gpl-2.0.html GNU General Public License (GPL)
- * @version		SVN: $Id: EmailHandler.php 12313 2013-09-15 21:14:35Z skenow $
+ * @category ICMS
+ * @package Messaging
+ * @subpackage Email
+ * @copyright (c) 2007-2008 The ImpressCMS Project - www.impresscms.org
+ * @license http://www.gnu.org/licenses/old-licenses/gpl-2.0.html GNU General Public License (GPL)
  */
 
-
 /**
  * load the base class
  */
-require_once ICMS_LIBRARIES_PATH . '/phpmailer/class.phpmailer.php';
-require_once ICMS_LIBRARIES_PATH . '/phpmailer/class.smtp.php';
+require_once ICMS_LIBRARIES_PATH . '/phpmailer/PHPMailer.php';
+require_once ICMS_LIBRARIES_PATH . '/phpmailer/SMTP.php';
+require_once ICMS_LIBRARIES_PATH . '/phpmailer/Exception.php';
 
 /**
  * Mailer Class.
  *
- * @author		Jochen B��nagel	<jb@buennagel.com>
- * @copyright	copyright (c) 2000-2003 The XOOPS Project (http://www.xoops.org)
+ * @author Jochen B��nagel <jb@buennagel.com>
+ * @copyright copyright (c) 2000-2003 The XOOPS Project (http://www.xoops.org)
  *
- * @category	ICMS
- * @package		Core
- * @subpackage	Mail
+ * @category ICMS
+ * @package Core
+ * @subpackage Mail
  */
-class icms_messaging_EmailHandler extends PHPMailer {
+class icms_messaging_EmailHandler extends PHPMailer\PHPMailer\PHPMailer {
```

Comment:
can you use `use PHPMailer\PHPMailer\PHPMailer; on top of the file` and keep `class icms_messaging_EmailHandler extends PHPMailer {` here? I think, It would look something better :|

Reply:
Thank you - I did make that change and tested on my server and it worked. I will adjust and commit.